README.md

```diff
@@ -132,20 +132,13 @@ Other ways of retrieving source code are "convenience" methods. For example, tag
 
 > Note: Airflow Summit 2020's "Production Docker Image" talk where context, architecture and customization/extension methods are [explained](https://youtu.be/wDr3Y7q2XoI).
 
-## Beyond the Horizon
+## Project Focus
 
-Airflow **is not** a data streaming solution. Tasks do not move data from
-one to the other (though tasks can exchange metadata!). Airflow is not
-in the [Spark Streaming](http://spark.apache.org/streaming/)
-or [Storm](https://storm.apache.org/) space, it is more comparable to
-[Oozie](http://oozie.apache.org/) or
-[Azkaban](https://azkaban.github.io/).
+Airflow works best with DAGs that are mostly static and slowly changing. When the DAG structure is similarly from one run to the next, it allows for clarity around unit of work and continuity. Other similar projects include [Luigi](https://github.com/spotify/luigi), [Oozie](http://oozie.apache.org/) and [Azkaban](https://azkaban.github.io/).
```

Comment:
We do not define DAG anywhere in this section. Should we use `workflows (DAGs)` in the first sentence?

Reply:
Should we add comparison to Apache NiFi too?